### DISCUSSION TITLE
How does `defer` impact the SEO performance?

### DISCUSSION THREAD

**Discussion by iamkd** [remix-run/remix#6579] [Q&A] [answered]
Hi! Does anyone know if deferred responses are parsed/evaluated correctly by the crawlers, or should we disable `defer` for bot requests to optimize the SEO performance?

**Comment by sergiodxa** [comment, marked as answer]
All templates detects if the request comes from a bot and wait until the whole data is read to send the HTML, so crawlers will get the complete HTML instead of receiving it with suspended parts.
